src/ViewModel/TextField.php

```diff
@@ -62,12 +64,14 @@ protected function __construct(
         $this->value = $value;
         $this->state = $state;
         $this->messageGroup = $messageGroup;
+        $this->formFieldInfoLink = $formFieldInfoLink;
     }
 
     public static function emailInput(
         FormLabel $label,
         string $id,
         string $name,
+        FormFieldInfoLink $formFieldInfoLink,
```

Comment:
Should be optional?

Reply:
We'll always need a privacy notice link by an email field, hence mandating the link here. The tradeoff is that the honeypot has to have it, even though it's meaningless in that context.

Should the mandatory nature of the field be imposed at a higher level, i.e. somewhere in `journal`?

Reply:
We should let the implementor handle it (since it's not required by the pattern).